src/operator/tvmop/op_module.cc

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ * Copyright (c) 2019 by Contributors
+ * \file op_module.cc
+ * \brief Invoke registered TVM operators.
+ * \author Yizhi Liu
+ */
+#if MXNET_USE_TVM_OP
+#include <tvm/runtime/packed_func.h>
+#include <tvm/runtime/registry.h>
+#include <tvm/runtime/c_runtime_api.h>
+#include <string>
+#include <vector>
+#include "op_module.h"
+
+using namespace tvm::runtime;
+
+namespace tvm {
+namespace runtime {
+
+void TVMOpModule::Load(const std::string &filepath) {
```

Comment:
Maybe need to load more information like version, backend_support, feature(avx512 or not) to help analysis.

Reply:
agree. maybe add later? haven't been very clear what/how to pass these information and how to use them
Currently as it is loading an AOT compiled module, avx instruction info won't help.

Reply:
For standard release, there's only one aot compiled module, so the avx512 info seems not useful. But to support multi-version of libtvmop, it's useful to know If the avx512 version is loaded or not.

Reply:
But avx512 flag is used during compiling operators. The binary code in `filepath` here already contains avx-enabled/disabled code.